### ISSUE TITLE
Toggling on mod makes AI-Shoujo unplayable because I can't toggle it off once I activate some cheats...

### ISSUE THREAD

**Issue by XenithXemaku** [ManlyMarco/IllusionCheatTools#18]
Hi, Xenith Xenaku here,

1. Every time I try to toggle this off in AI-Shoujo it simply refuses to and makes the game unplayable because of all the menus obstructing key items (such as the speech text, the main objective and the map). Sure toggling **_on_** the mod works as intended and the cheats work fine as far as I can tell.
2. Please at least add an option to close out the menus, because i can't _believe_ that there is no option for this just in case the above happens. The only way I found to close the menus... is by exiting the game, which is **_REALLY_** annoying!

**Comment by ManlyMarco** [comment]
Press the button again to hide the menu. Make sure to deselect any input boxes first and move the mouse outside of any mod windows. Make sure this and runtime editor are up to date.

**Comment by XenithXemaku** [comment]
> Make sure this and runtime editor are up to date.

1. According to the game, the installed version of AI_Cheat Tools is v2.9.0.0 (the ones I saw online are lower and nowhere **_near_** this). 
2. How do I check the runtime version?

> Press the button again to hide the menu. Make sure to deselect any input boxes first and move the mouse outside of any mod windows.

3. I tried that and it doesn't work, I press the button (in my case "C") and the menu pops up, I activate some cheats and try to deselect the menus afterwards, I press "C" again and it just refuses to close.
